source/LogbookPanel.cpp

```diff
@@ -51,6 +52,24 @@ namespace {
 	const string MONTH[] = {
 		"  January", "  February", "  March", "  April", "  May", "  June",
 		"  July", "  August", "  September", "  October", "  November", "  December"};
+
+	const string STORYLINES = "Storylines";
+
+void DisplayVector(const string &heading, const vector<string> &entries, 
```

Comment:
ident this function please

Reply:
Done, also added some missing blank lines.

Reply:
You don't need to be worried about small commits to fix things, so long as the thing keeps running its all good, many commits dont hurt
Saying this caus I assume you have this in a local commit still